app/src/main/res/menu/downloads_completed.xml

```diff
@@ -16,6 +16,12 @@
         android:title="@string/downloads_log_label"
         custom:showAsAction="always" />
 
+    <item
+        android:id="@+id/action_delete_downloads_played"
+        android:icon="@drawable/ic_delete_auto"
+        android:title="@string/delete_downloads_played"
+        custom:showAsAction="ifRoom" />
```

Comment:
I would never show it as an action icon, always in the overflow menu. I don't think users would understand from an icon what it does.